src/hct_mis_api/apps/payment/models.py

```diff
@@ -610,6 +725,29 @@ def excluded_beneficiaries_ids(self) -> List[str]:
         )
         return beneficiaries_ids
 
+    @property
+    def excluded_household_ids(self) -> List:
+        # TODO: moved from TP, maybe remove because we have excluded_beneficiaries_ids()
```

Comment:
we have 2 exclusions, on targeting level and on payment plan level, let's rename these properties to avoid confusion